### PR TITLE
Building the documentation pages

### PR DESCRIPTION
**Pull request by rpodgorney**

### CODE REVIEW

File: doc/content/documentation.menu.md [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+## Theory
 
-- [manuals/theory.md]
-- [manuals/user.md]
-- [Syntax](syntax/index.md)
-- [application_usage/peacock.md]
+- [Governing Equations](https://mooseframework.inl.gov/modules/porous_flow/governing_equations.html)
+- [Material Laws](https://mooseframework.inl.gov/modules/porous_flow/material_laws.html)
+- [Fluid Equations of State](https://mooseframework.inl.gov/modules/porous_flow/fluids.html)
+- [Geochemistry](https://mooseframework.inl.gov/modules/geochemistry/theory/index.html)
```
Comment (crswong888) [inline doc/content/documentation.menu.md:9]: These pages are already built by Falcon docs, and its better to link to the local content like:

```
- [modules/porous_flow/governing_equations.md]
- [modules/porous_flow/material_laws.md]
- [Fluid Equations of State](modules/porous_flow/fluids.md)
- [Geochemistry](modules/geochemistry/theory/index.md)
```

File: doc/content/documentation.menu.md [hunk 2]
```diff
@@ -1,32 +1,33 @@
...
 
 !col-end!
 
 !col! small=12 medium=6 large=4
 ## Development
 
-- [manuals/developer.md]
+- [Code Standards](https://mooseframework.inl.gov/sqa/framework_scs.html)
```
Comment (crswong888) [inline doc/content/documentation.menu.md:16]: Same thing here, use `- [Code Standards](sqa/framework_scs.md)`

File: doc/content/examples/index.md [hunk 3]
```diff
@@ -1,35 +1,19 @@
-!alert construction title=[Initialize Documentation #43 [!icon!link]](https://github.com/idaholab/falcon/issues/43) prefix=False
-All pages on this website are currently under development and do not yet serve as general documentation of the [FALCON repository](https://github.com/idaholab/falcon).
+!alert construction title=Develop Documentation prefix=False
+This page is currently under development and does not yet serve as general documentation of the [FALCON repository](https://github.com/idaholab/falcon).
 
-# Falcon Examples id=examples
+!media falcon/anim_elder_fe_lv1.mp4
```
Comment (crswong888) [inline doc/content/examples/index.md:4]: This should work once the [large media commit](https://github.com/idaholab/large_media/commit/06d3b39786410c388f8b47cc016ffd269e4e61cf) is pulled in, which needs to be done in the MOOSE repo.

**Review by crswong888 — CHANGES_REQUESTED** [review]
Everything looks good to me! just a few changes:

**Comment by rpodgorney** [comment]
@crswong888  just pushed revision with recommended changes.  Got errors on 

- [Geochemistry](modules/geochemistry/theory/index.md)

so kept it pointed at the url

**Pull request merged**